src/js/utils/api.js

```diff
@@ -26,6 +26,10 @@ function status(response) {
 }
 
 function json(response) {
+    if (response.status === 202 || response.status === 204) {
```

Comment:
Pretty sure it was throwing, but now I'm just trying to test it again and I can't figure a call we do an end up with a 204